src/distributions/weighted.rs

```diff
@@ -298,6 +297,38 @@ impl<W: AliasMethodWeight> Distribution<usize> for AliasMethodWeightedIndex<W> {
     }
 }
 
+impl<W: AliasMethodWeight> ::core::fmt::Debug for AliasMethodWeightedIndex<W>
+where
+    W: ::core::fmt::Debug,
+    super::Uniform<W>: ::core::fmt::Debug,
```

Comment:
You can `use core::fmt` already, and `Result` is already in the prelude.

Reply:
Fixed.